Ginger/Ginger/WindowExplorer/Common/WindowExplorerPage.xaml.cs

```diff
@@ -88,14 +92,15 @@ public enum eWindowExplorerPageContext
         }
 
         private eWindowExplorerPageContext mContext;
-
+        private ApplicationPOMModel mPOM;
         // We can open it from agents grid, or from Action Edit page with Action 
```

Comment:
All WindowExplorerPage changes which related to POM should be canceled and removed